host-bin/edit-chroot

```diff
@@ -360,7 +407,7 @@ for NAME in "$@"; do
             # Search for the alphabetically last tarball with src.
             # Dated tarballs take precedence over undated tarballs.
             for file in "$file."* "$file-"*; do
-                if [ ! -f "$file" ]; then
+                if [ ! -f "$file" -o "${file%part[0-9][0-9]}" != "$file" ]; then
```

Comment:
FAT32 limits the number of files in a directory to 65534.

So you get five digits now :)